Source/Shaders/Builtin/Constants/ellipsoidInverseRadii.glsl

```diff
@@ -0,0 +1,7 @@
+/**
+ * 0.1
```

Comment:
It looks like you copied this from the epsilon constants - I believe this should be a description of what this value is. For this, maybe it should say something like:

> The reciprocal of the radius of the WGS84 ellipsoid.